pkgs/applications/misc/slides/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "slides";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "maaslalani";
+    repo = "slides";
+    rev = "v${version}";
+    sha256 = "0gsjq8866rdbhgl8nmawx3iqm4ww26xlnybwl16a4mwvwcai0min";
+  };
+
+  vendorSha256 = "1wld36xi7ybl8k6m9kvszhrzr492k5yl87p5vc8vkx21bnblv1qp";
+
+  doCheck = false;
```

Comment:
Please add a comment why tests are disabled.

Reply:
Sorry, I don't think there was a reason. I enabled it, thanks!

Reply:
You didn't push that change.

Reply:
When I bumped to `0.2.0` then I did need the checks to be disabled so I added a comment.

Reply:
One last question: do we have the program that are required for the tests already in nixpkgs? You can add them with checkInputs. I think we should try enabling tests to make sure the package works correctly.

Reply:
Sorry about the late response, yes the programs are in `nixpkgs` my tests use bash, go, ruby, and python to verify that the different languages code can be executed if they are code blocks on the slides (I will probably remove these tests in a later version however, I can update this package when I remove those tests).

Reply:
I updated the `checkInputs` to match what is expected on the tests for now and turned the checks back on.